src/Microsoft.ApplicationInsights/Metrics/MetricsCollection.cs

```diff
@@ -63,6 +63,17 @@ public bool IsReadOnly
             return metric;
         }
 
+        /// <summary>@ToDo: Complete documentation before stable release. {799}</summary>
```

Comment:
Create a task for these so @MS-TimothyMothra can track it for the specific milestone. Also - please ensure documentation exists for `beta` as well, not only stable

Reply:
Done:
https://github.com/Microsoft/ApplicationInsights-dotnet/issues/817